http/NgapS3Credentials.h

```diff
@@ -24,30 +24,32 @@
 #ifndef HYRAX_GIT_S3CREDENTIALS_H
 #define HYRAX_GIT_S3CREDENTIALS_H
 
-#include "AccessCredentials.h"
 #include <string>
 
+#include "AccessCredentials.h"
+
+namespace http {
+
 class NgapS3Credentials : public AccessCredentials {
 public:
     // These are the string keys used to express the normative key names
     // for the credentials components.
-    static const std::string AWS_SESSION_TOKEN;
-    static const std::string AWS_TOKEN_EXPIRATION;
-    static const std::string BES_CONF_S3_ENDPOINT_KEY;
-    static const std::string BES_CONF_REFRESH_KEY;
-    static const std::string BES_CONF_URL_BASE;
+    static const char *AWS_SESSION_TOKEN;
```

Comment:
All of these key names should end in `_KEY` 
And we really need a better way to handle this - maybe char* is the right way but that seems like a step backwards to me.

Reply:
In this case, there is a #define in NgapS3Credentials.cc named AWS_SESSION_TOKEN_KEY and it has a different string value. I'm going to leave this one as it is.